code/game/machinery/Sleeper.dm

```diff
@@ -36,6 +36,10 @@
 	component_parts += new /obj/item/stack/cable_coil(null, 1)
 	RefreshParts()
 
+/obj/machinery/sleeper/Destroy()
+	go_out()
+	return ..()

```

Comment:
. = ..() is preferable to return ..() because of reasons i heard on the internet